masterlist.yaml

```
@@ -3545,6 +3545,14 @@ plugins:
       - crc: 0xA44BF120
         util: 'TES4Edit v4.0.4b'
 
+  - name: 'Lynges Dark Brotherhood Assassins-8654.esp'
```

It's not that the old entry is outdated, it's just that some browsers replace spaces with underscores when downloading files. This can be accounted for using a regex.

```suggestion
  - name: 'Lynges[ _]Dark[ _]Brotherhood[ _]Assassins-8654\.esp'
```